data/json/mutations/mutations.json

```diff
@@ -1143,7 +1143,7 @@
     "id": "ANTIWHEAT",
     "name": { "str": "Grain Intolerance" },
     "points": -2,
-    "description": "You have a rare allergy that prevents you from eating most types of grains such as wheat or oats.  It's possible for you to eat them, but you will suffer morale penalties and obtain less nutrition from them.",
+    "description": "You have a rare allergy that prevents you from eating most types of grains, such as wheat or oats.  It's possible for you to eat them, but you will suffer morale penalties and obtain less nutrition from them.",
```

Comment:
Oats?

Reply:
vOv The code only mentions wheat. I could cut oats from the description?

Reply:
That's what I'm thinking. If this is some variety of gluten intolerance, oat gluten is not close enough to stimulate a reaction in most people. (The main problem comes when wheat flour is used to help in cutting oats - manufacturers seem to have reduced this in recent years.)

Reply:
See how this strikes you: 
`You have a rare allergy that prevents you from eating wheat.  It's possible for you to eat wheat-based products, but you will suffer morale penalties and obtain less nutrition from them.`